multi_label_trainer_tfidf.ipynb

```diff
@@ -63,9 +63,10 @@
         "from sklearn.model_selection import train_test_split\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        print(f"Label(s): {', '.join([label for label in top_3_labels])}")
</pre><p> </p><p>"Predicted label(s)..." might be a better keyword.</p><p> </p><p>Also, at least add a single comment to explain what's going on in the prediction decoding. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>